modules/claims_api/spec/fixtures/v2/veterans/claims/claim_by_id_response.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
nit: mostly just to document my opinion here:
because this is a separate gem, i think specs within this project should mock the abstracted methods and not the actual bgs service responses. testing of the bgs service responses is for the gem to deal with, and this project shouldnt need to be aware of the underlying implementation of it's dependency.